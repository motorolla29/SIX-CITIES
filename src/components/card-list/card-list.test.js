import React from "react";
import renderer from "react-test-renderer";
import CardList from "./card-list";
import { OFFERS_DATA } from "../../mocks/offers";

it("Card list correctly renders after relaunch", () => {
  const tree = renderer.create(<CardList offersData={OFFERS_DATA} />).toJSON();

  expect(tree).toMatchSnapshot();
});
