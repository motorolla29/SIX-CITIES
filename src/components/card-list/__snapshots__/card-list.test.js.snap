// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card list correctly renders after relaunch 1`] = `
<div
  className="cities__places-list places__list tabs__content"
>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="https://picsum.photos/260/201"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            120
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              {
                "width": "93%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating: 
            3.2
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
        >
          Beautiful & luxurious studio at great location
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Private room
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="https://picsum.photos/260/199"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            220
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              {
                "width": "93%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating: 
            2.1
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
        >
          Wood and stone place
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Single Room
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="https://picsum.photos/260/198"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            400
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              {
                "width": "93%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating: 
            4.7
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
        >
          Nice, cozy, warm big bed house
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Place image"
          className="place-card__image"
          height="200"
          src="https://picsum.photos/260/203"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            155
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              {
                "width": "93%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating: 
            1.5
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
        >
          Place widh a view in the heart of the city
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Flat
      </p>
    </div>
  </article>
</div>
`;
